snapshot/vfs/vfs.go

```diff
@@ -310,8 +311,70 @@ func (fsc *Filesystem) Pathnames() iter.Seq2[string, error] {
 	}
 }
 
-func (fsc *Filesystem) GetEntry(path string) (*Entry, error) {
-	return fsc.lookup(path)
+func (fsc *Filesystem) GetEntry(entrypath string) (*Entry, error) {
+	if !strings.HasPrefix(entrypath, "/") {
```

Comment:
is this necessary ?

do we ever enter this without a prefix / ?

if that's the case, shouldn't it do more than just slap a / upfront ?

Reply:
it was in lookup, which in turns had it because the pre-btree code had something like it.  i guess it's just cargo culting from my part, and that it could be deleted.  i'll do a sweep.